app/assets/javascripts/application.js

```diff
@@ -1,4 +1,5 @@
 //= require components/autocomplete.js
+//= require components/reorderable-list.js
```

Comment:
I don't think we need this in here as well as admin_layout.js. If I check Collections Publisher it looks like both of those JS files are output for the more modern styled aspect of the app:

```

</footer>
 <script src="/assets/admin_layout-12fb822da5d9c5cdea69ce9669a3523aaba84d2e6f82aec041ec34caebe6f9fd.js"></script>
 <script src="/assets/application-aeca4222e3c68a2ca356f31412fad4f1349e6f313a92b8dc8e71c48994676ca2.js"></script>
 </body>
</html>
```

So I think we can remove this from this file and expect the component only to be used when admin_layout.js is included.